Google.PowerShell.IntegrationTests/BigQuery/BigQuery.GcbqProject.Tests.ps1

```diff
@@ -0,0 +1,24 @@
+﻿. $PSScriptRoot\..\BigQuery\GcbqCmdlets.ps1
+$project, $configName = Set-GCloudConfig
```

Comment:
This should be 
`$project, $zone, $oldActiveConfig, $configName = Set-GCloudConfig`
or 
`$_, $_, $oldActiveConfig, $configName = Set-GCloudConfig`
if you are never going to care about the default project or zone.